### DISCUSSION TITLE
pint is converting delta_degree_Celsius to degree_Fahrenheit

### DISCUSSION THREAD

**Discussion by akrherz** [hgrecco/pint#1904] [General]
I am using pint 0.23 and I thought for sure this was not working (as it shouldn't be) last year, but it appears to be doing this erroneous conversion now.

```python
>>> import pint
>>> ureg = pint.UnitRegistry()
>>> (ureg.Quantity(10, ureg.degC) - ureg.Quantity(5, ureg.degC)).to("degF")
<Quantity(-450.67, 'degree_Fahrenheit')>
```

**Comment by dalito** [comment]
This is a bug that existed since at least 0.6.1 (I did not go back further). The problem is that delta-units are not specially checked for when converting. I have a local fix and will submit a PR. Done --> #1905
